TestServer/README.md

```diff
@@ -1,7 +1,95 @@
-# Test Server
+# PrisonEscape Test Server
 
-To run a minecraft server there are a lot more files that are needed. However only the ones that have relevant changes are gonna be pushed to the repository.
+Welcome to the PrisonEscape Test Server repository! This repository contains all the necessary files to set up a Minecraft server for testing the PrisonEscape plugin. Please note that only files with relevant changes are included here. Additional setup and configuration may be required.
 
-## Map
+## Prerequisites
 
-When the first version of the map is ready a zip file will be added.
\ No newline at end of file
+To set up the Minecraft server, make sure you have the following:
+
+- Java installed on your system
+- Minecraft
+
+## Setup Instructions
+
+### 1. Download the Minecraft Server JAR
+
+1. Download the Paper server JAR [PaperMC Downloads version 1.20.4-462](https://api.papermc.io/v2/projects/paper/versions/1.20.4/builds/462/downloads/paper-1.20.4-462.jar).
+2. Place the downloaded JAR file in this directory.
+3. Run it.
+
+### 2. Initial Set-Up
+
+1. **Edit `eula.txt`**:
+   - Open the `eula.txt` file.
+   - Change `eula=false` to `eula=true`.
+   - Save the file.
+2. **Run `run.bat` to Start the Server**.
+3. **Stop the Server** after it starts successfully by typing `stop` in the console and pressing Enter.
+4. **Add the WorldEdit Plugin**:
+   - Download the latest WorldEdit JAR from [CurseForge](https://dev.bukkit.org/projects/worldedit).
+   - Place the downloaded JAR file in the `plugins` folder.
+5. **Install a World Management Plugin like Multiverse-Core**:
+   - Download the Multiverse-Core JAR.
```

Comment:
If you want to show the download page use this: https://dev.bukkit.org/projects/multiverse-core